.github/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+### What is the issue
+...
+
+### What does this PR fix and why was it fixed
+...
+
+### Checklist before you submit for review
```

Comment:
My 2 cents: I would just put the checklist, I do not find the previous two paragraphs very useful as they are kind of a duplicate of "Each commit has a meaningful description".

Reply:
Well - I think that it is still useful; actually we can see it is being used. That is helpful to have an overview of the purpose of the PR. That message will automatically become the commit message if one decides to squash/merge instead of rebase/merge